src/connectors/cloudtrail.py

```diff
@@ -0,0 +1,211 @@
+from runners.helpers import db, log
+from runners.helpers.dbconfig import WAREHOUSE
+
+FILE_FORMAT = """
+    TYPE = "JSON",
+    COMPRESSION = "AUTO",
+    ENABLE_OCTAL = FALSE,
+    ALLOW_DUPLICATE = FALSE,
+    STRIP_OUTER_ARRAY = TRUE,
+    STRIP_NULL_VALUES = FALSE,
+    IGNORE_UTF8_ERRORS = FALSE,
+    SKIP_BYTE_ORDER_MARK = TRUE
+"""
+
+CLOUDTRAIL_LANDING_TABLE = """
+(raw VARIANT,
+hash_raw NUMBER,
+event_time TIMESTAMP_LTZ(9),
+aws_region STRING,
+event_id STRING,
+event_name STRING,
+event_source STRING,
+event_type STRING,
+event_version STRING,
+recipient_account_id STRING,
+request_id STRING,
+request_parameters VARIANT,
+response_elements VARIANT,
+source_ip_address STRING,
+user_agent STRING,
+user_identity VARIANT,
+user_identity_type STRING,
+user_identity_principal_id STRING,
+user_identity_arn STRING,
+user_identity_accountid STRING,
+user_identity_invokedby STRING,
+user_identity_access_key_id STRING,
+user_identity_username STRING,
+user_identity_session_context_attributes_mfa_authenticated BOOLEAN,
+user_identity_session_context_attributes_creation_date STRING,
+user_identity_session_context_session_issuer_type STRING,
+user_identity_session_context_session_issuer_principal_id STRING,
+user_identity_session_context_session_issuer_arn STRING,
+user_identity_session_context_session_issuer_account_id STRING,
+user_identity_session_context_session_issuer_user_name STRING,
+error_code STRING,
+error_message STRING,
+additional_event_data VARIANT,
+api_version STRING,
+read_only BOOLEAN,
+resources VARIANT,
+service_event_details STRING,
+shared_event_id STRING,
+vpc_endpoint_id STRING)
+"""
+
+
+def create_connector(data):
```

Comment:
I think we're calling these `connection_options` and their type should be defined in the module, as well